packages/app/components/Header/MobileMenuList.tsx

```diff
@@ -0,0 +1,38 @@
+import Link from "next/link";
+import Blocks from "../../public/assets/images/blocks.svg";
+import Image from "next/image";
+
+interface MobileMenuListProps {
+  open: boolean;
+  toggle: () => void;
+}
+
+export default function MobileMenuList({ open, toggle }: MobileMenuListProps) {
+  if (!open) return null;
+
+  return (
+    <>
+      <div
+        className="fixed top-[72px] bottom-0 left-0 right-0 bg-gray-alpha-75"
+        onClick={toggle}
+      ></div>
+      <div className="w-full bg-surface-25 rounded-lg absolute top-[72px] left-0 border-b border-solid border-surface-75 gap-2 py-2 px-6">
```

Comment:
Wondering if we should have `72px` or just `72` somewhere as I'm seeing this value in various files/elements

Reply:
we can use tailwind top-20 which is 80px, and replace in all of this places.

But I'm okay with having the custom value in this 2 files. It's the height of the nav and here is to show after that nav.

I don't think will use that size any more. So adding to tailwind config is not a good idea.

Reply:
Nice point, forgot about that. That's why we are using tailwind. Thanks for bringing it up Diogo.